sdk/identity/azure-identity/src/samples/java/com/azure/identity/credential/JavaDocCodeSnippets.java

```diff
@@ -26,8 +26,8 @@ public final class JavaDocCodeSnippets {
     private String tenantId = System.getenv("AZURE_TENANT_ID");
     private String clientId = System.getenv("AZURE_CLIENT_ID");
     private String clientSecret = System.getenv("AZURE_CLIENT_SECRET");
-    private String username = "sampleuser";
-    private String password = "Samp1eP@ssw0rd";
+    private String username = "username";
+    private String fakePasswordPlaceholder = "fakeCredentialPlaceholder";
```

Comment:
is this naming enforced or we can rename to fakePassword ?

Reply:
Not enforced will change to fakePasswordPlaceholder